packages/yew-agent/src/hooks.rs

```diff
@@ -30,6 +30,67 @@ where
 ///
 /// Takes a callback as the only argument. The callback will be updated on every render to make
 /// sure captured values (if any) are up to date.
+///
+/// # Examples
+///
+/// ```
+/// use serde::{Deserialize, Serialize};
+/// use yew::prelude::*;
+/// use yew_agent::{use_bridge, UseBridgeHandle};
+///
+/// // This would usually live in the same file as your worker
+/// #[derive(Serialize, Deserialize)]
+/// pub enum WorkerResponseType {
+///     IncrementCounter,
+/// };
+///
+/// # use yew_agent::{HandlerId, Public, WorkerLink};
+/// # pub struct Worker {
+/// #   pub link: WorkerLink<Self>,
+/// # }
+///
+/// # impl yew_agent::Worker for Worker {
+/// #   type Input = ();
+/// #   type Output = WorkerResponseType;
+/// #   type Reach = Public<Self>;
+/// #   type Message = ();
+/// #
+/// #   fn create(link: WorkerLink<Self>) -> Self {
+/// #       Worker { link }
+/// #   }
+/// #
+/// #   fn update(&mut self, _msg: Self::Message) {
+/// #       // do nothing
+/// #   }
+/// #
+/// #   fn handle_input(&mut self, _msg: Self::Input, id: HandlerId) {
+/// #       self.link.respond(id, WorkerResponseType::IncrementCounter);
+/// #   }
+/// # }
```

Comment:
Checking the preview docs over on https://yew-rs-api--pr2722-spanishpear-use-brid-vrp7wa5n.web.app/next/yew_agent/fn.use_bridge.html this currently leads to three blank lines. To not include the whole `Worker` impl, the best I can imagine is something like

```rust
# mod my_crate {
# // the hidden stuff
# }
use my_crate::MyWorker; // <MyWorker as Worker>::Output == WorkerResponseType
// ...
        let bridge: UseBridgeHandle<MyWorker>
```

so as to not confuse the Worker trait and the user impl for MyWorker.

Reply:
I think I get what you mean - will have a go! :D